src/test/fuzz/addrman.cpp

```diff
@@ -96,31 +96,12 @@ class CAddrManDeterministic : public CAddrMan
             for (size_t j = 0; j < num_addresses; ++j) {
                 const auto addr = CAddress{CService{RandAddr(), 8333}, NODE_NETWORK};
                 const auto time_penalty = insecure_rand.randrange(100000001);
-#if 1
```

Comment:
I compared this branch with the `#else` branch, filled addrman with 9k addresses and did not observe any difference in performance. That is - now the high level `Add_()` + `Good_()` perform the same way as the hackish direct add to the tried table :racehorse:.